tests/test_other.py

```diff
@@ -6226,12 +6226,16 @@ def test_dash_s_link_flag(self):
     self.assertContained('hello, world!', self.run_js('a.out.js'))
 
   def test_dash_s_response_file_string(self):
-    create_file('response_file', '"MyModule"\n')
-    self.run_process([EMXX, test_file('hello_world.cpp'), '-s', 'EXPORT_NAME=@response_file'])
+    create_file('response_file.txt', 'MyModule\n')
+    create_file('response_file.json', '"MyModule"\n')
```

Comment:
is the newline needed in both cases? if so should we document that, as it seems like a pitfall? (but I hope it isn't needed)

Reply:
No, its not needed, but text files in general should always end in newlines.. (editors normally add one).. so including the newline is closer to what most uses in the wild will be like